float64 axis_1
float64 axis_2
bool button_1
bool button_2
bool button_3
bool button_4
bool button_5
bool button_6
bool button_7
bool button_8
